makefile

```
@@ -71,13 +75,17 @@ compile-pypi-amd64:
 compile-pypi-arm64:
 	docker run -it -v ${PWD}:/root/hostpwd/ hatchery-bootstrap-arm64 /root/hostpwd/compile.sh /root/hostpwd/py314.local.conf
 
-compile-aorg:
-	docker run -it -v ${PWD}:/root/hostpwd/ hatchery-bootstrap-aorg /root/hostpwd/compile-aorg.sh /root/hostpwd/py314.local.conf
+compile-aorg-amd64:
+	docker run -it -v ${PWD}:/root/hostpwd/ hatchery-bootstrap-aorg-amd64 /root/hostpwd/compile-aorg-new.sh /root/hostpwd/py314.local.conf
```

`compile-aorg-new.sh` missing, can use existing `compile-aorg.sh`.
was there a change intended on `compile-aorg-new.sh` ? 
```suggestion
	docker run -it -v ${PWD}:/root/hostpwd/ hatchery-bootstrap-aorg-amd64 /root/hostpwd/compile-aorg.sh /root/hostpwd/py314.local.conf
```